modules/apps/frontend-theme/kyoung-liferay-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key=" lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+    <title>${the_title} - ${company_name}</title>
+
+    <meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+    <#--  pt animation js for header -->
+    <script src="/o/kyoung-liferay-theme/js/pt.min.js"></script>

```

Comment:
Instead of calling the JS file directly, is there a way to include this via the files that are already being included in the theme's JS?

We are going to be including the unminified source code for this third party JS file (if needed) so we should leverage the Liferay code minifier whenever possible.  The JS code brought in via Liferay will be minified if we turn off the developer settings.